feconf.py

```diff
@@ -577,6 +577,7 @@ def get_empty_ratings():
 # TODO(sll): Add all other URLs here.
 ADMIN_URL = '/admin'
 ADMIN_ROLE_HANDLER_URL = '/adminrolehandler'
+ANALYTICS_DATA_URL = '/analytics'
```

Comment:
Everywhere you refer to "analytics", make it GOOGLE_ANALYTICS instead (or google_analytics for module names), so that we don't confuse it with the lesson analytics or other types of analytics.

Reply:
Okay